plugins/statistics.koplugin/main.lua

```diff
@@ -2819,11 +2788,9 @@ function ReaderStatistics:onReadingResumed()
     self._reading_paused_ts = nil
 end
 
-function ReaderStatistics:onReadSettings(config)
+function ReaderStatistics:onReaderReady(config)
     self.data = config:readSetting("stats", { performance_in_pages = {} })
-end
-
-function ReaderStatistics:onReaderReady()
+    self.doc_md5 = config:readSetting("partial_md5_checksum")
```

Comment:
May be mention it as `doc_md5 = nil` inside `Widget:extend{}`.
(Was wondering while reading from top to bottom whether you forgot to initialize it :))